gitops/components/nidhogg/overlays/nidhogg-config-patch.yaml

```diff
@@ -5,7 +5,7 @@ data:
       "taintNamePrefix": "nidhogg.uswitch.com",
       "taintRemovalDelayInSeconds": 10,
       "nodeSelector": [
-        "pelo.tech/multi-home-networking"
+        "kubernetes.io/os=linux"
```

Comment:
`on-demand-networking` and `metal` are the 2 node pools labeled with `pelo.tech/multi-home-networking` where we are expecting nidhogg to remove the startup taint `nidhogg.uswitch.com/kube-system.kube-multus-ds`?

Does it mean now Multus is going everywhere?

Reply:
I think it's two different things -
1. nidhogg controller can run anywhere and I've changed to it to run on the hostNetwork and tollerate anything so it's a basically guaranteed to work.
2. I think this node selector is for the config to choose where to look for the taintName and ds to be running and realistically it doesn't matter to us. so removing this limit gives us more flexibility without having to redeploy the config.

this allows us to
1. have a setup where we can have part of the nodes with multus and a custom network
2. have a setup where everything as multus and the same custom network

Reply:
https://pelotech.slack.com/archives/C05TEM6CE0Z/p1711988983895169?thread_ts=1711984332.335429&cid=C05TEM6CE0Z